WORKSPACE

```diff
@@ -135,3 +135,9 @@ git_repository(
 load("@io_bazel_rules_d//d:d.bzl", "d_repositories")
 
 d_repositories()
+
+git_repository(
+    name = "io_bazel_structure_test",
```

Comment:
`io_bazel` seems like the wrong namespace.  Why isn't this in the `repositories()` function?

Reply:
I was just stupidly pattern matching, but you're right it doesn't really belog here. moved